client/src/browser/components/BrowserWrapper.tsx

```diff
@@ -17,18 +22,45 @@ export const mapDispatchToProps = {
 
 type MergedProps = ReturnType<typeof mapStateToProps> & DispatchProps<typeof mapDispatchToProps>;
 
-const BrowserWrapper: React.SFC<MergedProps> = ({ isOpen, open }) => {
+const BrowserWrapper: React.SFC<MergedProps> = ({ isOpen, open, noOfDatasets, isVisible}) => {
     if (!isOpen) {
-        return (
-            <Button icon={true} labelPosition="left" onClick={open}>
-                <Icon name='add' />
-                Browse
-            </Button>
+        return ( 
+            <div>
+                <div style={{textAlign: 'center'}}>
+                    { (!noOfDatasets && !isVisible) ?
```

Comment:
There is another state that is important here: is there currently a dataset detection in progress?  This is important, as right now, if you open a dataset, the "empty state" flashes up, together with the "Detecting parameters for {path}" spinner, which is a bit distracting. Detection can also take a while, especially if your data is on a slower medium, like a network share. While detecting, the empty state should not be shown.

It looks like this:

![image](https://user-images.githubusercontent.com/5778/79213026-c79e3200-7e48-11ea-81e9-7637e5e7880e.png)

I included instructions in the screenshot for reproducing this state: you need to dispatch a `'DATASET_DETECT'` redux action, with any path as payload. This should trigger a perpetual `busy` state.